.appveyor.yml

```diff
@@ -1,21 +1,18 @@
 environment:
-  matrix:
-    - TOXENV: py27-cov
-    - TOXENV: py36-cov
+  global:
+    CIBW_SKIP: cp33-* cp34-* cp35-*
+    CIBW_TEST_REQUIRES: pytest pytest-cov
+    CIBW_TEST_COMMAND: "cd {project}\\tests && pytest --cov"
```

Comment:
maybe you could have `tox` as the only `CIBW_TEST_REQUIRES`, and again `tox` as the only `CIBW_TEST_COMMAND`. This way the CI and the local developer's test workflow is the same and we avoid duplication.

Reply:
Hm... will it pick up the built package?

Reply:
so you need to find out where cibuildwheel saves the built wheel package, then the CIBW_TEST_COMMAND will be `tox --installpkg $wheel`

this way tox will not install from source but will install the given wheel inside its own venv, then proceed to run the tests